exporter/prometheusexporter/accumulator.go

```diff
@@ -165,11 +165,13 @@ func (a *lastValueAccumulator) accumulateSum(metric pmetric.Metric, il pcommon.I
 
 	// Drop metrics with unspecified aggregations
 	if doubleSum.AggregationTemporality() == pmetric.AggregationTemporalityUnspecified {
+		a.logger.Debug(fmt.Sprintf("Dropping metric with unspecified aggregation: %s", metric.Name()))
```

Comment:
You should not use `fmt.Sprintf` to prevent unnecessary work when debug level is not enabled.
```suggestion
		a.logger.Debug("Dropping metric with unspecified aggregation", zap.String("metric", metric.Name())
```